sugargame/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '1.1'
+__version__ = '1.2'
```

Comment:
Btw, why aren't we porting this activity to Sugargame 1.3 again? I might be missing something

Reply:
Yes we need to, I will do it.